pyseq.py

```diff
@@ -80,6 +83,14 @@
 warnings.simplefilter('always', DeprecationWarning)
 
 
+def _natural_key(x):
```

Comment:
Separating this might be a good idea, but it will only work for file sequences with numerical pattern matching (see my groups branch https://github.com/rsgalloway/pyseq/tree/groups)